From ced8cf2fbfe411a969c73f4c8586046237c85b78 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 28 Nov 2019 17:22:50 +0800
Subject: [PATCH] arm64: rockchip_defconfig: add security configs

Sync with floral_defconfig
CONFIG_BUG_ON_DATA_CORRUPTION
CONFIG_SCHED_STACK_END_CHECK
CONFIG_DEFAULT_MMAP_MIN_ADDR to 32768

Change-Id: I11a8fc79a47da62e4f3df04da4614e6fcfc8e247
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 264308633689..08449364035a 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -110,6 +110,7 @@ CONFIG_PARTITION_ADVANCED=y
 CONFIG_CMDLINE_PARTITION=y
 # CONFIG_IOSCHED_DEADLINE is not set
 # CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS is not set
+CONFIG_DEFAULT_MMAP_MIN_ADDR=32768
 CONFIG_CMA=y
 CONFIG_ZSMALLOC=y
 CONFIG_NET=y
@@ -867,5 +868,7 @@ CONFIG_SOFTLOCKUP_DETECTOR=y
 CONFIG_DEFAULT_HUNG_TASK_TIMEOUT=10
 CONFIG_PANIC_TIMEOUT=5
 CONFIG_SCHEDSTATS=y
+CONFIG_SCHED_STACK_END_CHECK=y
 # CONFIG_DEBUG_PREEMPT is not set
 CONFIG_ENABLE_DEFAULT_TRACERS=y
+CONFIG_BUG_ON_DATA_CORRUPTION=y
-- 
2.35.3

